demisto_sdk/commands/format/update_mapper.py

```diff
@@ -67,6 +67,9 @@ def remove_non_existent_fields(self):
         """
         Remove non-existent fields from a mapper.
         """
+        if not self.id_set_file:
```

Comment:
please add logging, explaining what happens what the behavior is with and id_set, and without it